common/component/kafka/kafka.go

```diff
@@ -313,6 +316,12 @@ func (k *Kafka) getLatestSchema(topic string) (*srclient.Schema, *goavro.Codec,
 		return nil, nil, err
 	}
 
+	// measure the time taken to get the schema
```

Comment:
Why do we need this?

Reply:
just because originally I had some concerns about the time taken... I wish we could add a prometheus metric for serialization time but this was my closest proxy in case there are some future concerns about performance